pkg/operator/controller/canary/http.go

```diff
@@ -0,0 +1,119 @@
+package canary
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	routev1 "github.com/openshift/api/route/v1"
+
+	"github.com/tcnksm/go-httpstat"
+)
+
+const (
+	echoServerPortAckHeader = "x-request-port"
+)
+
+// probeRouteEndpoint probes the given route's host
+// and returns a bool indicating whether or not the request was
```

Comment:
Stale comment (no Boolean value is returned).

Reply:
fixed, thanks!